### ISSUE TITLE
"purple color" is not set correctly.

### ISSUE THREAD

**Issue by NBKRedSpy** [NBKRedSpy/BTShowMechAffinity#3]
Deployment count of 30 to 49 is using an invalid color of 282/255f.

```csharp
pilotColor.OverrideWithColor(new Color(282 / 255f, 59 / 255f, 73 / 255f)); //Purple
```

Change to a valid color.

**Comment by NBKRedSpy** [comment]
Gold was also mentioned to show up as red as well.

**Comment by NBKRedSpy** [comment]
Fixed by v1.1.0
